node-fs-readfile/cat.js

```diff
@@ -0,0 +1,15 @@
+
+var fs = require('fs');
+
+var filesToRead = process.argv;
+var lengthOfArgv = process.argv.length
+
+
+// console.log(fileToRead);
+
+for (var i = 2; i < lengthOfArgv; i++) {
+  fs.readFile(filesToRead[i], 'utf8', (err, data) => {
+    if (err) throw err;
+    console.log(data);
+  });
+}
```

Comment:
While this _starts_ the file reads in the same order, it doesn't guarantee they print out in the same order, since file reads happen in parallel and they may take different amounts of time:

---

![image](https://user-images.githubusercontent.com/16143771/128444258-71db0a0b-9e79-43bd-8c61-b59c4d766780.png)

---

In this example, `cunningham.txt` prints first, then `hipster-ipsum.txt`, then the other two (which are cut off from the screenshot), which is not the same order they were specified.